modules/persistent-storage-snapshots-restore.adoc

```diff
@@ -30,42 +34,14 @@ metadata:
     snapshot.alpha.kubernetes.io/snapshot: snapshot-demo <1>
 spec:
   storageClassName: snapshot-promoter <2>
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
The storage attribute specified is an example size of 1Gi but there should be a note stating to set the storage size to at least accommodate the original PV size.

Reply:
Added annotation: "Storage size for a restored snapshot must be large enough to accommodate the original PV size."